### PR TITLE
Add pricing to shipments

### PR DESCRIPTION
**Pull request by bion**
Also introduces a new type to the schema to represent quantities of money in any currency:
```graphql
type MoneyAmount {
  # Currency is a string holding an ISO-4217 currency code
  currency: Currency!
  quantityInMinorUnits: Int!
}
```

### CODE REVIEW

File: schema.graphql [hunk 1]
```diff
@@ -132,13 +135,36 @@ type Shipment {
...
+  singlePallet: MoneyAmount
+  halfPallet: MoneyAmount
+}
+
+input ShipmentPricingInput {
+  singlePallet: MoneyAmountInput
+  halfPallet: MoneyAmountInput
+}
+
+type MoneyAmount {
```
Comment (bion) [inline schema.graphql:156]: maybe this should just be called `Money`?

File: src/tests/helpers/index.ts [hunk 2]
```diff
@@ -26,7 +24,10 @@ async function createGroup(
 }
 
 async function createShipment(input: ShipmentCreateInput): Promise<Shipment> {
-  return await Shipment.create(input)
+  return await Shipment.create({
+    ...input,
+    pricing: input.pricing || undefined,
```
Comment (deammer) [inline src/tests/helpers/index.ts:29]: Question (non-blocking): why is this necessary?

File: schema.graphql (hunk 1 above)
Comment (deammer) [reply schema.graphql:156]: I think `MoneyAmount` describes it better

**Review by deammer — APPROVED** [review]
💵

File: src/tests/helpers/index.ts (hunk 2 above)
Comment (bion) [reply src/tests/helpers/index.ts:29]: 😭 because graphql codegen types non-required things as `Thing | null | undefined` and sequelize types non-required things as `Thing | undefined` 😭 😭 😭 😭 😭

Comment (bion) [reply src/tests/helpers/index.ts:29]: See commit message
